docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -338,11 +349,41 @@ and *W* is limited to 87 when there are 64 input channels.
 
 .. note:: 
 
-   The above limitations only apply to the new hardware 2D convolution operation. When possible, the Intel® GNA 
+   The above limitations only apply to the new hardware 2D convolution operation. For GNA 3.0 When possible, the Intel® GNA
    plugin graph compiler flattens 2D convolutions so that the second generation Intel® GNA 1D convolution operations 
    (without these limitations) may be used. The plugin will also flatten 2D convolutions regardless of the sizes if GNA 2.0 
    compilation target is selected (see below).
 
+
+Support for Convolutions since GNA 3.5
+--------------------------------------------------------------------------------------------------------------------------------------
+
+Staring from The Intel® GNA 3.5 convolution 1D are handled in different way than in GNA 3.0. Convolution have following limitations:
+
+============================  =======================  =================
+ Limitation                    Convolution 1D           Convolution 2D
+============================  =======================  =================
+Input height                   1                        1-65535
+Input Width                    1-65535                  1-65535
+Input channel number           1                        1-1024
+Kernel number                  1-8192                   1-8192
+Kernel height                  1                        1-255
+Kernel width                   1-2048                   1-256
+Stride height                  1                        1-255
+Stride width                   1-2048                   1-256
+Dilation height                1                        1
+Dilation width                 1                        1
+Pooling window height          1-1                      1-255
+Pooling window width           1-255                    1-255
+Pooling stride height          1                        1-255
+Pooling stride width           1-255                    1-255
+============================  =======================  =================
+
+
+Limitations for GNA 3.5 refers to the specific dimension. Whole scope of parameters is not fully supported,
```

Comment:
```suggestion
Limitations for GNA 3.5 refer to the specific dimension. Full range of parameters is not always supported,
```

Reply:
Here is where "the" is needed.  "Full range of parameters" should be "The full range of parameters".